src/lib/Nav.svelte

```diff
@@ -12,7 +13,7 @@
 				>
 			</li>
 
-			{#if $session.user}
+			{#if $user}
```

Comment:
I think under this new solution this evaluates as `true` all the time. 
```suggestion
			{#if $user.username}
```